docs/notebooks/LORIS_API_Part2_Python-script.ipynb

```diff
@@ -0,0 +1,2149 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "M_RHTKtPZ_G3"
+   },
+   "source": [
+    "# LORIS API Tour 2/2\n",
+    "\n",
+    "This tutorial contains basic examples in Python to demonstrate how to interact with the API.\n",
+    "To run this tutorial, click on `Runtime -> Run all`",
+    "This tutorial is also available as a Google colab notebook so you can run it directly from your browser. To access it, click on the button below: <a href=\"https://colab.research.google.com/github/aces/Loris/blob/main/docs/notebooks/LORIS_API_Part2_Python-script.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
the link https://colab.research.google.com/github/aces/Loris/blob/main/docs/notebooks/LORIS_API_Part2_Python-script.ipynb won't work until the PR is merged. For testing, use https://colab.research.google.com/github/spell00/Loris/blob/2020-07-20-jupyterApiPart2/docs/notebooks/LORIS_API_Part2_Python-script.ipynb